lib/src/layer/polyline_layer.dart

```diff
@@ -37,6 +40,7 @@ class Polyline {
     this.strokeCap = StrokeCap.round,
     this.strokeJoin = StrokeJoin.round,
     this.useStrokeWidthInMeter = false,
+    this.tag,
```

Comment:
Is there a reason we shouldn't use `key`? Or at the very least just accept any `Object?`?

Reply:
Yeah you are right. I think it would be better to pass an `Object?`. The tag should be used in case you want to specify the tap for one specific polyline. In my project we use Ids for polyline and trigger some event based on this. I think it is easier to handle an Object you choose constructing your polyline rather than some keys.